.pre-commit-config.yaml

```diff
@@ -232,6 +232,7 @@ repos:
 exclude: |
   (?x)^(
       MODULE.bazel.lock|
+      bazel/clang_tidy/.*\.patch|
```

Comment:
Maybe `bazel_clang_tidy` to avoid any ambiguity with a patch to `clang-tidy` itself?

Reply:
Done